test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -419,6 +427,7 @@ public class State extends CommonState
         private final List<Symbol> searchableNonPartitionColumns;
         private final List<Symbol> searchableColumns;
         private final List<Symbol> nonPkIndexedColumns;
+        private final boolean isCas;
```

Comment:
i actually plan to make this a new test class that extends, rather than add CAS at the lower level.  The main reason is mixing CAS with non CAS isn't deterministic